appgate/__main__.py

```diff
@@ -219,7 +219,7 @@ async def run_k8s(args: OperatorArguments) -> None:
         start_entity_loop(
```

Comment:
Don't you need to pass the the api version here somehow?
It needs to be used here I think: https://github.com/appgate/sdp-operator/blob/57d04961a6bf1d08c4205fccae4bb7a4e94ebae5/appgate/appgate.py#L117